internal/ccsmp/ccsmp_core.go

```diff
@@ -362,12 +362,12 @@ func (session *SolClientSession) solClientSessionSubscribeReplyTopicWithFlags(to
 	return handleCcsmpError(func() SolClientReturnCode {
 		cString := C.CString(topic)
 		defer C.free(unsafe.Pointer(cString))
-		// This is not an unsafe usage of unsafe.Pointer as we are using correlationId as data, not as a pointer
+		// This is not an unsafe usage of unsafe.Pointer as we are using dispatchID and correlationID as data, not as pointers
 		return C.SessionReplyTopicSubscribeWithFlags(session.pointer,
```

Comment:
same, fix comment or remove.  There is no unsafe.Pointer question here.  And all the following spots too...